pkg/testutil/env/env.go

```diff
@@ -139,13 +137,6 @@ func CreateActiveEnvForTest() *Env {
 	return NewEnv(runtime.NewSchemeBuilder(api.AddToScheme))
 }
 
-func RunCode(m *testing.M, e *Env) int {
```

Comment:
Since all of the e2e tests use the same setup, perhaps we could add something like this (no biggie though):

```go
func WithActiveEnv(fn func (*Env)) {
  env := CreateActiveEnv()
  defer env.Stop()

  fn(env.Start())
}
```

Reply:
`defer` needs to be called in each Test* func itself - with this it'll get called right after WithActiveEnv returns. I think I can make this a little bit cleaner though - please hold

Reply:
Oh wait, I think I see what you're getting at, lemme see if I can make that work

Reply:
To be clear, I was thinking the tests would use this. Like:

```go
func TestMyThing(t *testing.T) {
  ...
  ...
  env.WithActiveEnv(func(env *env.Env) {
    ...tests here
  })
}
```

Honestly, it's not a big deal. If that feels over-engineered we can skip it for now

Reply:
Yeah, I think that feels a little much, but if env setup gets more complex I think doing this would be good. Let's just keep it as-is for now?